MapleServer2/Database/DatabaseContext.cs

```diff
@@ -496,10 +545,12 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<GameEvent>(entity =>
             {
                 entity.HasKey(e => e.Id);
-                entity.Property(e => e.Type);
+                entity.Property(e => e.Type).HasConversion<string>();
```

Comment:
Is this really needed?

Reply:
Considering most of these game events would be done through SQL (I think?), I thought it'd be useful to know what the type is by string rather than a digit.
So no it's not _REALLY_ needed

Reply:
Oh ok, nvm then